src/app/resourcebundles/data/consumption/en.properties

```diff
@@ -1148,4 +1148,18 @@ frmelmnts.lbl.useThis = Use this:
 frmelmnts.lbl.noResultFoundFor = No results found for "{query}" from
 frmelmnts.lbl.content.COPYRIGHT = COPYRIGHT
 messages.emsg.m0076= No data available to download 
+frmelmnts.desktop.lbl.importNewFile = Import new file
```

Comment:
We need to add the bundles to phraseapp for translations and before adding new check if they already exists